agasc/supplement/magnitudes/mag_estimate.py

```diff
@@ -881,7 +881,7 @@ def get_agasc_id_stats(agasc_id, obs_status_override=None, tstop=None):
     stats['mean_corrected'] = np.nan
     stats['weighted_mean'] = np.nan
 
-    star = get_star(agasc_id, date=all_telem['times'][0])
+    star = get_star(agasc_id)
```

Comment:
Should this have `use_supplement=False`? Otherwise the `MAG_ACA` below will be from the supplement if available.

Reply:
I thought we'd captured that as a separate issue but I guess I forgot to do so.

Reply:
This is now #106